src/features/WordTable/index.tsx

```diff
@@ -30,27 +30,38 @@ const WordTable = ({
   cards,
   moveCards,
   deleteCards,
+  flipCards,
 }: {
   cards: Card[];
   moveCards: (cardIds: string[], folderId?: string) => void;
   deleteCards: (cardIds: string[]) => void;
+  flipCards: (cardIds: string[]) => void;
 }) => {
   const { activeFolder } = UseFolderContext();
   const [activeCardIds, setActiveCardsIds] = useState<string[]>([]);
   const [searchInput, setInput] = useState("");
+  const [filter, setFilter] = useState("rec");
 
   //Search
   const fuse = new Fuse(cards, {
     keys: ["front.text", "back.text"],
   });
 
-  const filteredCards = !searchInput.length
+  var filteredCards = !searchInput.length
```

Comment:
oh why a `var`?

Reply:
why does it matter?

Reply:
`var` was the very old way of declaring js variables. The main difference is that `let` has stricter scoping, which makes it less prone to bugs.

Reply:
In most cases, it won't matter, but I think it's generally good practice to stick with `let`.